tests/conftest.py

```diff
@@ -435,6 +436,33 @@ def blockstore(backend_store, fixtures_customization):
     return config
 
 
+@pytest.fixture(autouse=True)
+def persistent_mockup(
+    fixtures_customization: dict[str, Any]
+) -> Generator[None, None, Callable[[], None] | None]:
+    if fixtures_customization.get("real_data_storage", False) or fixtures_customization.get(
+        "alternate_workspace_storage", False
+    ):
+        test_toggle_local_db_in_memory_mock(False)
+        yield None
+
+    else:
+        test_toggle_local_db_in_memory_mock(True)
+        # No database should be in use while we clear the mock. Hence it's
+        # better to do the clear as part of the init given autouse fixtures
+        # run first.
+        test_clear_local_db_in_memory_mock()
+
+        yield test_clear_local_db_in_memory_mock
+
+
+# `persistent_mockup` is autouse, hence a test only have it explicitly has dependency to be
+# able to use it clear function. Here we just re-expose this function with a better name ;-)
+@pytest.fixture
+def clear_persistent_mockup(persistent_mockup):
+    return persistent_mockup or (lambda: None)
```

Comment:
Would not it be simple to just rename `persistent_mock` to `clear_persistent_mock` ?
(and make it always return a Closure)

Reply:
yes that was my initial intent, but I didn't like the fact the name of the function didn't match at all it body. So introducing this dummy fixture make it more readable.

Reply:
Ok so